BlackJackets-ui/src/pages/VenueAdd.jsx

```diff
@@ -20,7 +21,8 @@ export default function VenuePage() {
       capacity != "" &&
       email != "" &&
       location != "" &&
-      phone != ""
+      phone != "" &&
+      video != ""
```

Comment:
I think this could be removed in the case that a Venue doesn't have a video to upload when they register